src/app/interactions/ripple/ripple-sample-3/ripple-sample-3.component.scss

```
@@ -1,3 +1,4 @@
 .sample-button{
     width: 7rem;
-}
\ No newline at end of file
+    margin: 10px;
```

Make sure that paddings and margins are divisible by 8. We base our designs on a 8x8 grid.
```suggestion
    margin: 8px;
```